uber/templates/preregistration/paid_preregistrations.html

```diff
@@ -34,12 +34,20 @@
     <p>
       Congratulations! Your registration was successful.
     </p>
+      {% if total_cost > 0 %}
     <p>
       Your credit card was successfully charged for
       <strong>${{ total_cost }}</strong>, and you will receive a confirmation
       email sent to you shortly. If you paid for other registrations, those
       attendees will get their own confirmation email.
     </p>
+      {% else %}
+          <p>
+          Woah! You got a free badge{% if preregs|length > 1 %}s{% endif %}.
```

Comment:
This is what this message could look like:

"Congratulations! Your registration was successful. Woah! You got a free badges!"

You can see two problems there: first of all, we tell you that you've registered and then we... basically tell you again. We can just chop off the entire first line. Secondly, the text always says 'a free badge' even if you got multiple badges, and you've just edited whether or not there's an 's'. This should be fixed.